frontend/packages/db-structure/src/parser/prisma/parser.ts

```diff
@@ -157,4 +157,38 @@ function normalizeConstraintName(constraint: string): ForeignKeyConstraint {
   }
 }
 
+function convertToPostgresColumnType(
```

Comment:
Can you please specify in code comments that we are following this document?
https://www.prisma.io/docs/orm/reference/prisma-schema-reference#model-field-scalar-types

Reply:
Sorry about that! I've added a comment to the reference.
https://github.com/liam-hq/liam/pull/572/commits/6c16e00647db190b6b37a0cdb597fd55fa6504ed